eng/scripts/Language-Settings.ps1

```diff
@@ -5,6 +5,58 @@ $packagePattern = "*.zip"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/python-packages.csv"
 $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=container&comp=list&prefix=python%2F&delimiter=%2F"
 
+function Get-AllPackageInfoFromRepo ($serviceDirectory)
+{
+  $allPackageProps = @()
+  $searchPath = "sdk/*/*/setup.py"
+  if ($serviceDirectory)
+  {
+    $searchPath = "sdk/${serviceDirectory}/*/setup.py"
+  }
+  Push-Location $RepoRoot
+  $allSetupProps = $null
+  try
+  {
+    pip install packaging==20.4 -q -I
+    $allSetupProps = python "eng\scripts\get_package_properties.py" -s $searchPath
+  }
+  catch
+  {
+    # This is soft error and failure is expected for python metapackages
+    LogError "Failed to get all package properties"
+  }
+  Pop-Location
+
+  foreach ($line in $allSetupProps)
+  {
+    $setupInfo = ($line -Split ",").Trim("()")
```

Comment:
I know this was my suggestion but looking at it a little more does it even work? Given that split returns a list? If it does work then perhaps go ahead and move the other trim chars here as well to avoid the duplication below.

Reply:
From a quick local test it does appear to work so I  would go ahead and move the other trim chars to here as well.